skimage/morphology/convex_hull.py

```diff
@@ -56,7 +56,7 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
     # the starting or ending pixel of a row or column.  This vastly
     # limits the number of coordinates to examine for the virtual hull.
     if ndim == 2:
-        coords = possible_hull(image.astype(np.uint8))
+        coords = possible_hull(np.ascontiguousarray(image.astype(np.uint8)))
```

Comment:
I think maybe you should use the dtype parameter in the ascontiguousarray
https://docs.scipy.org/doc/numpy/reference/generated/numpy.ascontiguousarray.html#numpy-ascontiguousarray

```suggestion
        coords = possible_hull(np.ascontiguousarray(image, dtype=np.uint8))
```

Reply:
I added this suggestion, as well as some tests for 3D arrays.